printing/printer.ml

```diff
@@ -828,9 +830,24 @@ let pr_goal_by_id ~proof id =
   try
     Proof.in_proof proof (fun sigma ->
       let g = Evd.evar_key id sigma in
-      pr_selected_subgoal (pr_id id) sigma g)
+      pr_selected_subgoal None (pr_id id) sigma g)
   with Not_found -> user_err Pp.(str "No such goal.")
 
+(** print a goal identified by the goal id as it appears in -emacs mode.
+ sid should be the Stm state id corresponding to proof.
+ Used to support the Prooftree tool in Proof General. (https://askra.de/software/prooftree/). *)
```

Comment:
Beware of the indentation / formatting of the comment.

Reply:
What is the convention?  The code is not consistent on this.

Reply:
Most OCaml projects are following `ocamlformat`, I think in this case would be breaking at 80 columns and spaces on the left.